include/inviwo/core/datastructures/bitset.h

```diff
@@ -143,6 +143,13 @@ class IVW_CORE_API BitSet : public Serializable {
      */
     bool isStrictSubsetOf(const BitSet& b) const;
 
+    /**
+     * replace the bitset with the contents of \p b, returns true if modified
```

Comment:
I think comments should start with a capital letter. And I would skip the empty row below.